azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/redux/middleware/handler/CallingMiddlewareActionHandler.kt

```diff
@@ -486,8 +498,8 @@ internal class CallingMiddlewareActionHandlerImpl(
                         store.dispatch(CallingAction.IsTranscribingUpdated(false))
                         store.dispatch(CallingAction.IsRecordingUpdated(false))
                         store.dispatch(ParticipantAction.ListUpdated(HashMap()))
-                        store.dispatch(CallingAction.StateUpdated(CallingStatus.NONE))
-                        if (store.getCurrentState().callState.operationStatus == OperationStatus.SKIP_SETUP_SCREEN) {
+                        if (store.getCurrentState().callState.operationStatus == OperationStatus.SKIP_SETUP_SCREEN
```

Comment:
nit: Should write these conditions inverted, i.e.
OperationStatus.SKIP_SETUP_SCREEN == store.getCurrentState().callState.operationStatus

1) Easier to read especially if there is multiple.
2) More null safe habit, i.e. 
(NonNull == Null) == False
 (Null == NonNull) == Null Pointer Exception.
 So constants on the left avoids a NPE situation.

Reply:
As per discussion we will fix for GA.